index.js

```diff
@@ -0,0 +1,57 @@
+module.exports = AsMVT;
+
+const tilelive = require("@mapbox/tilelive");
+const url = require("url");
+const zlib = require("zlib");
+const { Pool } = require("pg");
+
+function AsMVT(uri, callback) {
+  var parsedUrl = url.parse(uri, true);
+  const functionName = parsedUrl.query.function || "gettile";
+  this._query = "select " + functionName + "($1, $2, $3)";
+  this.pool = new Pool({
+    user: parsedUrl.query.user || process.env.PGUSER,
+    host: parsedUrl.query.host || process.env.PGHOST,
+    database: parsedUrl.query.database || process.env.PGDATABASE,
+    password: parsedUrl.query.password || process.env.PGPASSWORD,
+    port: parsedUrl.query.port || process.env.PGPORT
+  });
```

Comment:
It might not be in the scope of this module, but verifying that the postgres connection parameters are valid by testing the pool connection might be prudent.